TrafficCapture/nettyWireLogging/src/main/java/org/opensearch/migrations/trafficcapture/netty/ConditionallyReliableLoggingHttpRequestHandler.java

```diff
@@ -26,6 +27,8 @@ public ConditionallyReliableLoggingHttpRequestHandler(IChannelConnectionCaptureS
     @Override
     protected void channelFinishedReadingAnHttpMessage(ChannelHandlerContext ctx, Object msg, DefaultHttpRequest httpRequest) throws Exception {
         if (shouldBlockPredicate.test(httpRequest)) {
+            ctx.channel().attr(AttributeKey.valueOf("isBlocking")).set(true);
```

Comment:
Please make isBlocking a final static constant.  Keep in mind that this is not the only time that flush() will be called (the serializer may do it on its own).

Reply:
We should poke around, and this will be largely subject to change shortly as the HTTP decoder is rewritten, but the first line/headers of a message may be present, or could be present, after they've been processed.